session.go

```diff
@@ -552,6 +583,15 @@ func DialWithInfo(info *DialInfo) (*Session, error) {
 	if info.PoolLimit > 0 {
 		session.poolLimit = info.PoolLimit
 	}
+
+	if info.MinPoolSize > 0 {
```

Comment:
I think this checks should be reversed to return an error if MinPoolSize or MaxIdleTimeMS are <0